pkg/restapi/vc/operation/operations.go

```diff
@@ -742,6 +745,18 @@ func (o *Operation) createDIDUniRegistrar(keyType, signatureType string,
 		return "", "", "", fmt.Errorf("selected key not found %s", selectedKeyID)
 	}
 
+	if strings.Contains(identifier, "did:v1") {
```

Comment:
why do we need special case for v1?

Reply:
To fetch the keys based on purpose - AssertionMethod (for issuer) and Authentication (for holder.)

Reply:
but why is v1 special?